docker-build-release/ldc-builder-linux-x86_64/Dockerfile

```diff
@@ -22,20 +22,20 @@ ENV BUILD_ROOT=/build
 RUN mkdir ${BUILD_ROOT}
 WORKDIR ${BUILD_ROOT}
 
-# The 12.04 system GCC is too old for building recent LLVM versions, so grab a
+# The 14.04 system GCC is too old for building recent LLVM versions, so grab a
 # backport from the "unofficial-official" toolchain test PPA.
-RUN apt-get -y update && apt-get -yq install python-software-properties && \
+RUN apt-get -yq install software-properties-common && \
```

Comment:
> Also, not breaking things that already work should be a good starting point for any PR.

Regressions can always creep in as we know. And no, I don't recall any 404s, but only recently started using Linux more often. Will re-add the update, next time please just fix it yourself with an explanatory comment, that's probably more efficient if one's sure it's a regression.
Just to elaborate on my motivation: I got tired of the 3 consecutive `apt-get update` during testing.